src/components/NavigationSection.svelte

```diff
@@ -74,26 +73,26 @@
       return {
         busy: false,
         closed: true,
-        valve: false
+        isFetching: true
```

Comment:
Yet, it seems to works, but w/ my current setup I can't install the settings app, so the requests are executed quickly, and it _may_ mask some bugs. But yep, go to test it in prod :+1: